test/Cruzo1155Factory.ts

```diff
@@ -15,6 +17,8 @@ describe("CruzoMarket", () => {
   let owner: SignerWithAddress;
 
   const serviceFee = 300;
+  const rawVaultFuncSignature = RAW_VAULT_FUNCTION_SIGNATURE
```

Comment:
Meaningless assignment, you can just use the imported `RAW_VAULT_FUNCTION_SIGNATURE` value directly